plugins/repository-azure/qa/microsoft-azure-storage/build.gradle

```diff
@@ -71,6 +71,7 @@ testClusters.integTest {
         // in a hacky way to change the protocol and endpoint. We must fix that.
         setting 'azure.client.integration_test.endpoint_suffix',
                 { "ignored;DefaultEndpointsProtocol=http;BlobEndpoint=http://${azureStorageFixture.addressAndPort }" }
+        setting 'thread_pool.repository_azure.max', '1'
```

Comment:
perhaps randomize this to a value between 1 and 10 using `project.testSeed`?

Reply:
Ah I didn't know that kind of randomizing was a thing :) thanks => added it.